tests/sentry/backup/test_models.py

```diff
@@ -217,6 +223,53 @@ def test_organization(self):
         self.create_organization(owner=user)
         return self.import_export_then_validate()
 
+    @targets_models(Incident, Organization, AlertRule)
```

Comment:
I changed this slightly to be `@targets(mark(Foo, Bar))` going forward (it will be easier to test for coverage this way). Do you mind making the same change here (and for the tests that landed in the previous PR, like Actor etc. - sorry that I missed it on that review!)